contrib/go/src/python/pants/contrib/go/tasks/go_protobuf_gen.py

```diff
@@ -75,7 +77,12 @@ def execute_codegen(self, target, target_workdir):
 
     outdir = os.path.join(target_workdir, 'src', 'go')
     safe_mkdir(outdir)
-    target_cmd.append('--go_out={}'.format(outdir))
+    protoc_plugins = self.get_options().protoc_plugins + list(target.protoc_plugins)
```

Comment:
Hmm, I'm not sure what I think the right thing to do is. It's currently moot since grpc is the only plugin in existence, as far as I can tell. 

FWIW, javac/scalac plugins are additive: https://github.com/benjyw/pants/blob/1ec7de8edd46827fa63e15b5d8aede88ac42d5f2/src/python/pants/backend/jvm/tasks/jvm_compile/jvm_compile.py#L556